src/core/directives/google-map-marker.ts

```diff
@@ -71,6 +71,11 @@ export class SebmGoogleMapMarker implements OnDestroy,
   iconUrl: string;
 
   /**
+   * Whether to automatically open the info window when the marker is clicked.
```

Comment:
Change to: Whether to automatically open the child info window when the marker is clicked.